org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/BuildWorkspaceHandler.java

```diff
@@ -61,6 +62,13 @@ public BuildWorkspaceStatus buildWorkspace(boolean forceReBuild, IProgressMonito
 				return BuildWorkspaceStatus.CANCELLED;
 			}
 			projectsManager.cleanupResources(projectsManager.getDefaultProject());
+			IProject[] projects = ResourcesPlugin.getWorkspace().getRoot().getProjects();
```

Comment:
Could extract this part as an individual method?

Reply:
https://github.com/eclipse/eclipse.jdt.ls/blob/bf8db8fa9fc09c8d0aaec6370560e8eb846ae36d/org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/ProjectUtils.java#L83